### DISCUSSION TITLE
Size area filters per camera

### DISCUSSION THREAD

**Discussion by davefx** [blakeblackshear/frigate#4767] [General]
I have false-positive problems, and I'm trying to filter them about use the `min_area`  and `max_area` filters.

My problem is with some night-vision false-positives with a big size. I'd like to use the `max_area` filter for them, but my problem is that detections that would be too big for one of my cameras are totally OK for other ones.

Can I set up different area-size filters for every camera in the configuration?

**Comment by blakeblackshear** [comment]
Yes. You can set them at the camera level. The config reference tells you which options can be set at the camera level too. Almost every configuration option is available globally and per camera.

**Comment by davefx** [comment]
Good to know. I just checked https://docs.frigate.video/configuration/index/
and this is missing from the sample file. Maybe it would be a good idea to
add an example, or at least add a little comment saying that this is
possible.

Best regards,
--
David Marín Carreño ***@***.***>

**Reply by NickM-27** [reply]
It is there, that's exactly what

```yaml
# Optional: Object configuration
# NOTE: Can be overridden at the camera level

```

Means
